tests/libvmi/cloudinit.go

```diff
@@ -55,19 +55,22 @@ func WithCloudInitNoCloudNetworkData(data string) Option {
 }
 
 // WithCloudInitConfigDriveData adds cloud-init config-drive user data.
-func WithCloudInitConfigDriveData(userData string, b64Encoding bool) Option {
+func WithCloudInitConfigDriveData(userData string) Option {
 	return func(vmi *v1.VirtualMachineInstance) {
 		addDiskVolumeWithCloudInitConfigDrive(vmi, cloudInitDiskName, v1.DiskBusVirtio)
 
 		volume := getVolume(vmi, cloudInitDiskName)
-		if b64Encoding {
-			encodedData := base64.StdEncoding.EncodeToString([]byte(userData))
-			volume.CloudInitConfigDrive.UserData = ""
-			volume.CloudInitConfigDrive.UserDataBase64 = encodedData
-		} else {
-			volume.CloudInitConfigDrive.UserData = userData
-			volume.CloudInitConfigDrive.UserDataBase64 = ""
-		}
+		volume.CloudInitConfigDrive.UserData = userData
+	}
+}
+
+// WithCloudInitConfigDriveDataBase64 adds cloud-init config-drive with base64 encoded user data.
+func WithCloudInitConfigDriveDataBase64(userData string) Option {
```

Comment:
This function is not used (because this functionality is not tested).
@0xFelix [suggested](https://github.com/kubevirt/kubevirt/pull/10329#discussion_r1305670597) to remove it.
So its now removed.